### PR TITLE
app-forensics/libewf new maintainer. Version bump.

### PR DESCRIPTION
**Pull request by CanuteTheGreat**

### CODE REVIEW

File: app-forensics/libewf/libewf-20171104.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#IUSE="debug python unicode"
+IUSE="bfio debug ewf +fuse +ssl static-libs +uuid unicode zlib"
+
+DEPEND="
+	sys-libs/zlib
+	bfio? ( =app-forensics/libbfio-0.0.20120425_alpha )
+	fuse? ( sys-fs/fuse:= )
+	uuid? ( sys-apps/util-linux )
+	ssl? ( dev-libs/openssl:0= )
+	zlib? ( sys-libs/zlib )"
```
Comment (Dessa) [inline app-forensics/libewf/libewf-20171104.ebuild]: you are already unconditionally depending on zlib

File: app-forensics/libewf/libewf-20171104.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (mgorny) [inline app-forensics/libewf/libewf-20171104.ebuild]: Please use EAPI 6.

File: app-forensics/libewf/libewf-20171104.ebuild [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+AUTOTOOLS_AUTORECONF=1
```
Comment (mgorny) [inline app-forensics/libewf/libewf-20171104.ebuild]: Those two variables don't do anything. Please look critically at your own work.

File: app-forensics/libewf/libewf-20171104.ebuild [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		$(use_enable unicode wide-character-type)
+		$(use_with zlib)
+		# autodetects bzip2 but does not use
+		--without-bzip2
+		$(use_with bfio libbfio)
+		$(use_with ssl openssl)
+		$(use_with uuid libuuid)
+		$(use_with fuse libfuse)
+	)
+	autotools_src_configure
```
Comment (mgorny) [inline app-forensics/libewf/libewf-20171104.ebuild]: No such thing.

File: app-forensics/libewf/libewf-20171104.ebuild [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+src_install() {
+	autotools_src_install
```
Comment (mgorny) [inline app-forensics/libewf/libewf-20171104.ebuild]: Likewise.

**Pull request closed without merging**